AppConnectSwift/CreateAccountViewController.swift

```diff
@@ -45,8 +46,15 @@ class CreateAccountViewController: UIViewController {
     }
     
     func showAlert(title: String, message: String) {
+        var okAction = UIAlertAction(title: "OK", style: UIAlertActionStyle.Default) {
+            UIAlertAction in
+            NSOperationQueue.mainQueue().addOperationWithBlock({
+                self.performSegueWithIdentifier("CreateAccountSuccess", sender: nil)
+            })
+        }
```

Comment:
Alternatively if the method is only used once in the controller, maybe it could be renamed to something more specific, like 'accountCreatedSuccessfully()`  and continue to handle both the alert creation and the segue.